roles/app/templates/project/managed.py

```diff
@@ -115,7 +115,14 @@ def INSTALLED_APPS(self):  # noqa
     }
 
     # Email
-    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+    if {{ USE_SMTP }}:
```

Comment:
if should be in jinja templates not in python right?

Reply:
oh well it'll work too but if True / if False looks strange. We can fix this later though.

Reply:
yes, it looks strange, but this is set at deploy time anyway
The option is to set the backend as a variable and keep username etc in the setting anyway even if they are not used